### PR TITLE
Document some object checking

### PR DESCRIPTION
**Pull request by tobiasvl**

### CODE REVIEW

File: src/constants/entities.asm [hunk 1]
```diff
@@ -55,7 +55,7 @@ ENTITY_BOOMERANG                     equ $01
 ENTITY_BOMB                          equ $02
 ENTITY_HOOKSHOT_CHAIN                equ $03
 ENTITY_MAGIC_ROD_FIREBALL            equ $04
-ENTITY_ENTITY_LIFTABLE_ROCK          equ $05
+ENTITY_LIFTABLE_ROCK          equ $05
```
Comment (daid) [inline src/constants/entities.asm:58]: whitespace mismatch

File: src/constants/gfx.asm [hunk 2]
```diff
@@ -166,6 +166,7 @@ OBJECT_CLIFF_TOP            equ $4D
 OBJECT_BUSH                 equ $5C
 OBJECT_WEATHER_VANE_BASE    equ $5E
 OBJECT_WELL                 equ $61
+OBJECT_CACTUS               equ $69
```
Comment (daid) [inline src/constants/gfx.asm:169]: If cactus and spikes share the same check and object ID, maybe it makes sense to just have a "OBJECT_HURT_TILE" instead?

File: src/constants/memory/wram.asm [hunk 3]
```diff
@@ -2940,7 +2940,7 @@ wSwitchableObjectAnimationStage::
   ds 1 ; D6F8
 
 ; Unlabeled
-wD6F9::
```
Comment (daid) [inline src/constants/memory/wram.asm:2943]: Maybe this is to indicate if you are walking on a raised switchblock?

**Review by daid — APPROVED** [review]

File: src/constants/memory/wram.asm (hunk 3 above)
Comment (tobiasvl) [reply src/constants/memory/wram.asm:2943]: Yes, that seems likely. It's also used one place to play `NOISE_SFX_FOOTSTEP` when Link transitions to standing on the ground, ie. when falling down from a raised switchblock.

**Comment by kemenaran** [comment]
LGTM with the comments addressed.

(Also, @tobiasvl, just to clarify: when you request two reviewers, are you expecting validation from the two of them? As far as I'm concerned, one reviewer approval is enough: if it's approved by someone else, feel free to merge it, of course :) )

**Review by daid — APPROVED** [review]

**Comment by tobiasvl** [comment]
> (Also, @tobiasvl, just to clarify: when you request two reviewers, are you expecting validation from the two of them? As far as I'm concerned, one reviewer approval is enough: if it's approved by someone else, feel free to merge it, of course :) )

No, I just request the people I feel are most likely to have comments for the PR, so they notice it; I'll freely merge when one of them has approved, since that's what the repo is set up to allow :)

**Pull request merged**